RELEASE-LIST.md

```diff
@@ -35,6 +35,8 @@ Once all of that is in place, we're ready to go.
 
 - Hallway test the zip.
 
+  - Is the CKEditor in `views/shared/javascripts/dist/production/ckeditor`?
```

Comment:
@doana can you elaborate a bit on this addition? Have you run into CKEditor being missing?